bin/cleanup.sh

```diff
@@ -4,3 +4,8 @@
 for namespace in istio-system istio-control istio-control-master istio-ingress istio-telemetry; do
```

Comment:
Can we label all iop-installed namespaces with a istio:something label ? I would expect at some point we'll run the control plane and other components in namespaces derived from the PR ( to parallelize )

Reply:
We see your point, but don't think this should be part of this PR. We think this would require other changes as well, right?